### DISCUSSION TITLE
Couldnt get boot from SATA ssd working

### DISCUSSION THREAD

**Discussion by Bahazzie** [Joshua-Riek/ubuntu-rockchip#587] [General]
Hi people, with armbian Bookworm I succeeded getting this instructing to work:

https://www.reddit.com/r/OrangePI/comments/10iq625/guide_how_to_boot_orange_pi_5_from_sata_m2_ssd/

However no luck for Ubuntu from here...any ideas? Mind you it's a SATA ssd, not nvme!

Bas

**Comment by Bahazzie** [comment]
Never mind got it working...

**Reply by hakotani-o** [reply]
Hi Bahazzie.
Please add overlays=orangepi-5-sata.

**Comment by Bahazzie** [comment]
Awesome! I stumbled on JR RK3588 because of Geekbench6 results that were just a tad higher than the rest. Slick, smooth, sharp, fast!

Thanks a lot guys. I have 2 RK3588 boxes, one running Armbian with a lot of Docker stuff. And the OPI5 now running JR Linux :-)

Bas

**Comment by Bahazzie** [comment]
Geekbench6 multicore 3100, I am very pleased ;-)

**Comment by Bahazzie** [comment]
First time ever I saw supersmooth 1080 and even 4K playback on the Costa Rica 4K video on ANY of my RK3588 boxes, really! Very impressive.
